src/tests/address_test.ts

```diff
@@ -36,6 +36,109 @@ describe("Address", () => {
     });
   });
 
+  describe(".listTransactions()", () => {
```

Comment:
```suggestion
  describe("#listTransactions", () => {
```

`#listTransactions` refers to an instance method, whereas `.listTransactions` is used to denote a class method